### ISSUE TITLE
Tests for variance homogeneity

### ISSUE THREAD

**Issue by victor-moreno** [jonathon-love/moretests#1]
Hi Jonathon,
I think Levene's test is not enough because it is not always sensitive to large differences in variances. I have a teaching example where a variance ratio of 2 is not significant with Levene's test.
I would like to have the F test for t-tests and Bartlett's test for One-way ANOVA.

I think the moretests module might be a good place for them, but I can develop a new module if you prefer. In that case, I will copy the ideas of moretests.

Victor

**Comment by jonathon-love** [comment]
adding them to `moretests` sounds great! so this would add additional tables to both of these analyses?

jonathon

**Comment by victor-moreno** [comment]
Yes, I suppose these tests will require an extra table in each analysis.
Enlarging current table would require also changing the heading that now shows (Levene's) and I don't know it it can be done.

Victor

**Comment by jonathon-love** [comment]
hey,

sorry, wasn't sure if you were wanting a response to this. it's certainly trivial to add code to change the table heading, but it might be worth adding a new table, and hiding the old one (the same way i do with the normality tests).

cheers

jonathon

**Comment by victor-moreno** [comment]
Thanks, I will try to use just one table to make output more compact when I have the time to code this.
Victor
